data/avro-schema/src/main/resources/avro/net/corda/data/flow/state/checkpoint/PipelineState.avsc

```diff
@@ -20,6 +20,14 @@
       "type": ["null", "net.corda.data.ExceptionEnvelope"],
       "default": null,
       "doc": "Used for platform generated errors reported back to user code."
+    },
+    {
+      "name": "cpks",
+      "type": {
+        "type": "array",
+        "items": "net.corda.data.crypto.SecureHash"
+      },
+      "doc": "Array of stored cpks from the Virtual Node."
```

Comment:
These are hashes of the CPKs right?  Might be better to call them cpkHashes/cpkChecksums or something like that to not confuse them with actual CPKs. The doc should be updated to reflect that too,